### DISCUSSION TITLE
How to make it less resource heavy?

### DISCUSSION THREAD

**Discussion by lokman2k5** [aunetx/blur-my-shell#662] [Q&A]
I have blur enabled for all windows and it's being too heavy on the CPU since I only have an integrated graphics card (thinkpad) and temperatures sometimes go up to 80-85, is there a way to improve this? like some settings or something to install?

**Comment by aunetx** [comment]
Hello, unfortunately I don't have a very satisfying answer for you... I won't be able to make the applications blur less resources heavy until I implement the static blur for them, but even then it still will be quite heavy :(

Maybe you can try to deactivate the "no artifacts" option if you have it activated in the "other" option tab, but apart from this the only thing I can recommend is disabling the applications blur (or at least activate it for some specific apps only, and activate the "opaque focused window" in order not to make the blur be recalculated when you work on a window)

**Reply by lokman2k5** [reply]
Thanks for the quick response.
I do not have no artifacts on and enabling opaque focused window would stop the current window from looking so beautiful. How can I help speed up the process of implementing the static blur for applications? I am currently learning how to program gnome extensions
